internal/resources/otelcollector/rbac.go

```diff
@@ -112,6 +112,32 @@ func makeMetricGatewayClusterRole(name types.NamespacedName, kymaInputAllowed bo
 		})
 	}
 
+	if k8sClusterReceiverAllowed {
+		clusterRules := []rbacv1.PolicyRule{{
+			APIGroups: []string{""},
+			Resources: []string{"events", "namespaces/status", "nodes", "nodes/spec", "pods", "pods/status", "services", "replicationcontrollers", "replicationcontrollers/status", "resourcequotas", "namespaces", "pods"},
```

Comment:
Why is `pods` listed twice here?
Also, can we have the same order of the resources as in the documentation [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver#rbac) just to make it easier for comparison:?

Reply:
done